doc/reference/interactive_console.rst

```diff
@@ -120,3 +125,22 @@ call was ``console.set_default_output('lua')``.
                 | :code:`...`
 
             -   :code:`\x81 \xa3 \x6b \x65 \x79 \x01`
+
+.. _interactive_console-shortcuts:
+
+Keyboard shortcuts
+------------------
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 1
+
+        *   - Keyboard shortcut
+            - Effect
+
+        *  - ``CTL+C``
+           - Discard current input with the ``SIGINT`` signal in the console mode and
+             jump to a new line with a default prompt.
```

Comment:
But it shutdowns tarantool if a command is executed currently (when we don't wait for a user input).